swatch-contracts/ct/java/api/ContractsSwatchService.java

```diff
@@ -41,6 +44,21 @@ public class ContractsSwatchService extends SwatchService {
   private static final String RESET_DATA_ENDPOINT = ENDPOINT_PREFIX + "/rpc/reset/%s";
   private static final String CONTRACTS_ENDPOINT = ENDPOINT_PREFIX + "/contracts";
   private static final String SUBSCRIPTIONS_ENDPOINT = ENDPOINT_PREFIX + "/subscriptions";
+  private static final String GET_SKU_ENDPOINT =
+      "/api/rhsm-subscriptions/v1/subscriptions/products/{product_id}";
+  private static final String TERMINATE_SUB_ENDPOINT =
+      ENDPOINT_PREFIX + "/subscriptions/terminate/{subscription_id}";
+  private static final String PRODUCT_TAGS_ENDPOINT =
+      ENDPOINT_PREFIX + "/offerings/{sku}/product_tags";
+
+  public ContractsSwatchService() {
+    withProperty(
+        "quarkus.arc.exclude-types", "com.redhat.swatch.splunk.CountingSplunkErrorCallback");
+    withProperty("quarkus.analytics.disabled", "true");
```

Comment:
This is a build time property and configuring it in here, it does not have any effects when running the test in an ephemeral environment.